src/index.html

```diff
@@ -37,7 +37,8 @@
   <meta property="og:title" content="Just Arrived"/>
   <meta property="og:description" content="We give newcomers in Sweden a chance to enter the labor market"/>
   <meta property="og:image" content="/assets/images/logo.png"/>
-  <meta property="og:locale" content="sv"/>
+  <meta property="og:locale" content="sv_SE"/>
+  <meta property="fb:app_id" content="2287595228132792"/>
```

Comment:
BTW, I'm using [the same](https://github.com/justarrived/justarrived.github.io/blob/dev/_includes/head.html#L29) `fb:app_id`  on https://github.com/justarrived/justarrived.github.io is that the correct way of using it?